### ISSUE TITLE
new version really messed up configs just not show only those loliX

### ISSUE THREAD

**Issue by vajeeking** [ForlaxPy/openbullet-1.2#29]

**Comment by vajeeking** [comment]
yeah can u just not add that force update, and put it like optional, cause if u messed up new update it need roll backs u first version without update,,, problems just too many, like now configs gone cant see those or like that.

**Comment by shdf2** [comment]
I agree, forcing people to update when they don't want is not a good idea. We must keep control of our OB. if an update is reported to be bugged we must be able to stay with our previous version, **PLEASE** REMOVE THAT AUTOMATIC UPDATE !!!!
